inst/extdata/database/tedfs/Tech/Electric Arc Furnace.csv

```diff
@@ -1,17 +1,17 @@
 mode,reheating,variable,reference_variable,period,component,value,uncertainty,unit,reference_value,reference_unit,comment,source,source_detail
-*,*,CAPEX,Output Capacity|Steel Liquid,2013,#,184.0,,EUR_2013,1.0,t/a,,Wortler13,
-*,*,CAPEX,Output Capacity|Steel Liquid,2018,#,184.0,,EUR_2013,1.0,t/a,"This reference assumes the value provided by Wörtler et al., 2013.",Vogl18,
+*,*,CAPEX,Output Capacity|Steel Liquid,2013,#,184.0,,EUR_2010,1.0,t/a,,Wortler13,Exhibit 8
+*,*,CAPEX,Output Capacity|Steel Liquid,2018,*,184.0,,EUR_2010,1.0,t/a,"This reference assumes the value provided by Wörtler et al., 2013.",Vogl18,
```

Comment:
Because otherwise, [this line](https://github.com/PhilippVerpoort/posted/blob/9b84e34fc4a1e891fc88abcd929a56cbe36493b8/inst/extdata/database/tedfs/Tech/Electric%20Arc%20Furnace.csv#L7) stops working, as it is only a component of the OPEX Fixed relative (it doesn't see the CAPEX value defined with a `#` in `component`. My understanding is that POSTED interprets '#' as "total" and `*` as "applies to each individual component" ?

Reply:
Yes correct, the definition of `#` and `*` is as you stated it (though I think it still needs better documentation).

Ah. That's unfortunate. Because if you do it as you suggest, you would get this CAPEX entry for every possible value of the component field. Since it's not a coded value, it just picks up all values it can find in the column.


So running
```
DataSet('Tech|Electric Arc Furnace').aggregate(period=2030, agg=['mode', 'reheating'], source='Vogl18').query(f"variable.str.contains('CAPEX')")
```
will give you
```
                                            component  source   variable                        region  period      unit  value  
0                                                   #  Vogl18   Tech|Electric Arc Furnace|CAPEX  World    2030  USD_2005  216.6  
7                                              Labour  Vogl18   Tech|Electric Arc Furnace|CAPEX  World    2030  USD_2005  216.6  
10  Operation and maintenance incl. refractory lin...  Vogl18   Tech|Electric Arc Furnace|CAPEX  World    2030  USD_2005  216.6
```
So you get three copies of if this row: for the sum of all, for the labour, and for other O&M cost. I guess that's not what we want.

If you additionally aggregate over `component`, you get three times the value you wanted to get. So running
```
DataSet('Tech|Electric Arc Furnace').aggregate(period=2030, agg=['mode', 'reheating'], source='Vogl18').query(f"variable.str.contains('CAPEX')")
```
will give you
```
   source                         variable region  period      unit  value
0  Vogl18  Tech|Electric Arc Furnace|CAPEX  World    2030  USD_2005  649.8
```

I had implemented but never really tested that option to put `*` in a component-type field column. Right not, I'm not even sure what the use-case would be. But clearly it's not what you had intended.

It is in fact quite hard to get that OPEX entry in line 7 to be interpreted correctly. The way that component-type fields are implemented in POSTED right now, makes this quite hard. The variable mapping would not be only between variables but across different field values.

As an easy fix for now, I suggest that we calculate `OPEX Fixed` manually and mention this in the comment column. Any chance you could quickly make that change?